### PR TITLE
Add config values using env vars

### PR DESCRIPTION
**Pull request by Loki-Afro**
supply configuration values using -e like `-e CLAMD_CONF_StreamMaxLength=3000M` to add/replace them in the config for clamd or `FRESHCLAM_CONF_xxx` respectably for freshclam

inspired by https://github.com/mko-x/docker-clamav/blob/136fd83e5a7bad2e63fc87497be45d11eb838e94/debian/buster/envconfig.sh#L17-L23

unfortunately only possible in privileged mode as one can and should not modify the configs as clamd user

fixes gh-27

### CODE REVIEW

**Comment by Loki-Afro** [comment]
some  additional notes

- i wasn't sure where to change the entry point scripts if only the unstable or all versions .. thus i changed them all because you support them all (?) and i didn't saw any mentioning in the documentation on how to use for any version restrictions
- i needed to change the original code from mko-x slightly since here `sh` is used instead of `bash`  - hope you are fine with that :)

**Comment by Loki-Afro** [comment]
hey @rsundriyal what do you think? :)

File: clamav/1.0/alpine/scripts/docker-entrypoint.sh [hunk 1]
```diff
@@ -17,6 +17,23 @@ fi
 # Assign ownership to the database directory, just in case it is a mounted volume
 chown -R clamav:clamav /var/lib/clamav
 
+# configure freshclam.conf and clamd.conf from env variables if present
+env | grep "^CLAMD_CONF_" | while IFS="=" read -r KEY VALUE; do
+    TRIMMED="${KEY#CLAMD_CONF_}"
+
+    grep -q "^$TRIMMED " /etc/clamav/clamd.conf && \
```
[the remark this answers is not in the document]
Comment (rsundriyal) [reply clamav/1.0/alpine/scripts/docker-entrypoint.sh:24]: Config files have variables commented thus it is not able to find anything with current pattern. Adding "#" makes it work. 

Similar changes need to be done in other places too.

**Review by rsundriyal — CHANGES_REQUESTED** [review]
- Fix the regex pattern
- Fix the duplicate code for 1.4 release

**Comment by rsundriyal** [comment]
> hey @rsundriyal what do you think? :)

The code looks fine. I have requested the changes which I have tested too. 

Please add the changes.

**Comment by rsundriyal** [comment]
@Loki-Afro In case the notifications are missed. I need your feedback on it. 
Please take a look.

**Comment by micahsnyder** [comment]
@Loki-Afro Can you please take a look at @rsundriyal's recommendations?  

If you don't have time to work on it anymore let us know and we'd be happy to make the necessary changes ourselves so we can merge the PR.

**Comment by micahsnyder** [comment]
Replaced by #68

**Pull request closed without merging**